scheds/rust/scx_rustland/src/main.rs

```diff
@@ -264,14 +264,15 @@ impl<'a> Scheduler<'a> {
         let init_page_faults: u64 = 0;
 
         // Low-level BPF connector.
+        let nr_online_cpus = topo.span().weight() + 1;
```

Comment:
FYI, there's a small bug in topo where it computes span, that I'm about to send a fix for. Once that's addressed, you shouldn't need to do + 1 here.  I'll merge this and update your code here as part of that fix.

Reply:
> FYI, there's a small bug in topo where it computes span, that I'm about to send a fix for. Once that's addressed, you shouldn't need to do + 1 here. I'll merge this and update your code here as part of that fix.

I actually seemed like a bug, but I wasn't sure, thanks for clarifying it!

Reply:
https://github.com/sched-ext/scx/pull/191 should fix it